05-typedb-cluster/02-kubernetes.md

```diff
@@ -38,14 +41,40 @@ Depending on the deployment method you choose, next steps to perform the deploym
 
 **Use this mode if your an application resides within the same Kubernetes network.**
 
+Encryption can be enabled for this mode. Generated certificates need to be valid for `*.<helm-release-name>`,
+so for deployment named `typedb-cluster`, certificate needs to be valid for `*.typedb-cluster` hostnames.
+To generate them, execute these commands:
 
-Deploying TypeDB Cluster in non-exposed mode means it would only be accessible from within the same Kubernetes cluster. To do it, execute the command:
+```
+$ mkcert -cert-file rpc-certificate.pem -key-file rpc-private-key.pem "*.typedb-cluster"
+$ curl https://repo.vaticle.com/repository/meta/create-encryption-mq-key.tar.gz -s | tar -xzvf -
```

Comment:
Let's replace the instruction for downloading the encryption (MQ) tool: tell the user to download the distribution.

We'll update the instruction again when we decide where and how to package it.

Reply:
@lolski addressed in https://github.com/vaticle/docs/pull/523/commits/a7b67989eb484ba49459bad6747d3d01ebf83291